core/logging/fileLogging.go

```diff
@@ -0,0 +1,152 @@
+package logging
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"sync"
+	"time"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go-logger/redirects"
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+const minFileLifeSpan = time.Second
+const defaultFileLifeSpan = time.Hour * 24
+
+var log = logger.GetOrCreate("core/logging")
+
+// FileLogging is able to rotate the log files
+type FileLogging struct {
```

Comment:
make this lowercase

Reply:
Ok, will add an interface for it. Need the pointer in structs.go

Reply:
done